CMakeLists.txt

```diff
@@ -677,11 +685,6 @@ else()
 	target_compile_definitions(torrent-rasterbar PUBLIC TORRENT_DISABLE_DHT)
 endif()
 
-# Boost
-find_public_dependency(Boost REQUIRED COMPONENTS system)
-target_include_directories(torrent-rasterbar PUBLIC ${Boost_INCLUDE_DIRS})
```

Comment:
This and the next line were here for a reason: every new boost release breaks imported targets in `FindBoost.cmake`. Please revert this change.